### PR TITLE
Move variant's down arrow and font tweak

### PR DESCRIPTION
**Pull request by NoSloppy**

### CODE REVIEW

File: style_editor.html [hunk 1]
```diff
@@ -99,13 +99,12 @@
...
 
 /* Variant and Alt adjusters */
-
 .variant-alt-container {
   background: #6ADA;
   border: 1px solid black;
   font-family: "Helvetica Neue",Helvetica,Arial,sans-serif;
-  font-size: 15px;
-  padding: 2px;
+  font-size: .9em;
```
Comment (profezzorn) [inline style_editor.html:106]: It seems weird to define a font size in em (which is the font size)

Comment (NoSloppy) [reply style_editor.html]: yeah. changed to percentage.

**Comment by profezzorn** [comment]
This now has conflicts that needs to be resolved.

**Comment by NoSloppy** [comment]
> This now has conflicts that needs to be resolved.

Just resubmitting

**Pull request closed without merging**